lib/nim.coffee

```diff
@@ -111,16 +117,17 @@ module.exports =
       nimSuggestExe: fixSystemPath(atom.config.get('nim.nimsuggestExecutablePath') or 'nimsuggest')
       nimExe: fixSystemPath(atom.config.get('nim.nimExecutablePath') or 'nim')
       nimSuggestEnabled: atom.config.get 'nim.nimsuggestEnabled'
+      nimCustomArgs: parseArguments(atom.config.get('nim.nimCustomArguments') or '')
```

Comment:
I probably could have handled this more elegantly, but I wasn't sure what value Atom provides by default.